### PR TITLE
Logical properties

### PR DESCRIPTION
**Pull request by jamesnw**
An alternate feature set from this would be something like-

- Logical border (perhaps split into -color, -style, -width, etc)
- Logical clear
- Logical float
- Logical inset
- Logical min and max size
- Logical padding

And then have physical counterparts for each. 

In looking at how people are talking about this feature, it seemed that people were more interested in Logical properties all together, in knowing that they could use `-block-start` instead of `-top` (in LTR, TTB writing mode) across properties.

### CODE REVIEW

File: features/logical-properties.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: Logical properties
+description: The logical CSS properties control borders, size, margin, and padding with directions and dimensions relative to the writing mode. For example, in a left to right, top to bottom writing mode, `block-end` refers to the bottom. Also known as flow relative.
```
Comment (autonome) [inline features/logical-properties.yml:2]:
```suggestion
description: CSS logical properties control borders, size, margin and padding with directions and dimensions relative to the writing mode. For example, in a left to right, top to bottom writing mode, `block-end` refers to the bottom. Also known as flow relative.
```

Comment (autonome) [reply features/logical-properties.yml:2]: Minor grammatical disambiguation fix.

File: features/logical-properties.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+spec: https://drafts.csswg.org/css-logical-1/
+caniuse: css-logical-props
+group: css
+# Picked to match caniuse- clear and float came 2 years later, but don't warrant separate features.
```
Comment (autonome) [inline features/logical-properties.yml:6]:
```suggestion
# Picked to match Caniuse. Clear and float came 2 years later, but don't warrant separate features.
```

**Review by autonome — APPROVED** [review]
+1, logical properties are documented as a single feature across many guides, blogs, etc.

File: features/logical-properties.yml (hunk 1 above)
Comment (jamesnw) [reply features/logical-properties.yml]: Thanks for the review!

**Review by ddbeck — APPROVED** [review]
This is great. Thank you!

**Pull request merged**